source/reference/about-mongosync.txt

```diff
@@ -0,0 +1,131 @@
+.. _about-mongosync:
+
+========================
+About ``mongosync``
+========================
+
+The ``mongosync`` binary is the primary process used in {+c2c-product-name+}. 
+``mongosync`` migrates data from one cluster to another and keeps the 
+clusters in continuous sync. You can use ``mongosync`` to create dedicated 
+analytics, development, or testing clusters that mirror your production 
+environment. 
+
+``mongosync`` enters different states depending on the requests it receives. 
+The current ``mongosync`` state determines which API operations you can run. 
+For more information on ``mongosync`` states, see 
+:ref:`mongosync States <c2c-states>`.
+
+.. note:: 
+
+   Most state transitions happen as the result of an API call. The exceptions 
+   are:
+
+   - The transition from ``COMMITTING`` to ``COMMITTED``
+   - The transition from ``REVERSING`` to ``RUNNING``
+
+How ``mongosync`` Works
+------------------------
+
+Initial Connection
+~~~~~~~~~~~~~~~~~~~
+
+``mongosync`` must create an initial connection to the source and destination 
+clusters before it can start to sync data. For more information, see 
+:ref:`c2c-connecting`.
+
+When ``mongosync`` connects to the source and destination clusters, it is in the 
+:ref:`IDLE <c2c-state-idle>` state.  
+
+During Sync 
+~~~~~~~~~~~~
+
+The :ref:`start <c2c-api-start>` endpoint begins the 
+:ref:`RUNNING <c2c-state-running>` state. 
+
+If you start a new sync, ``mongosync`` syncs data to the destination cluster. 
+If you make subsequent writes to the source cluster, ``mongosync`` applies them 
+to the destination cluster.  
+
+For every collection on the source, ``mongosync`` creates a corresponding 
+collection on the destination. 
+
+``mongosync`` temporarily alters some collection characteristics during 
+the ``RUNNING`` state and changes these back to their source 
+cluster characteristics during the ``COMMIT`` state. For more information, see
+:ref:`Commit Alters <c2c-commit-alters>`.
```

Comment:
This may also be too technical for this page

Reply:
I added this part because Jian and Rohan mentioned this during the meeting as something to explain. I will leave this conversation unresolved so they can take a look during the external review.